crons/leaderboards.js

```diff
@@ -1,20 +1,34 @@
 const fs = require('fs/promises');
-const { createPool, sql } = require('slonik');
-const { sleep } = require('./utils/sleep');
-const accountCreationSql = require('./queries/account-creation.sql');
-const scoreSql = require('./queries/score-calculate.sql');
+const { sql } = require('slonik');
+const { sleep } = require('../utils/sleep');
+const accountCreationSql = require('../queries/account-creation.sql');
+const scoreSql = require('../queries/score-calculate.sql');
 
 class LeaderboardCache {
-  constructor(cachePoolConnectionString, indexerPoolConnectionString) {
-    this.cachePool = createPool(cachePoolConnectionString);
+  constructor(statsGalleryCachePool, indexerCachePool, environment) {
+    this.cachePool = statsGalleryCachePool;
 
-    this.indexerPool = createPool(indexerPoolConnectionString);
+    this.indexerPool = indexerCachePool;
+
+    this.environmentVar = environment;
```

Comment:
Why not just use `process.env`?

Reply:
The design I am aiming for is for this layer to not be concerned with the layer of environment variables. Also, it would be easier to test because we could just dependency inject an environment mock if we start doing test coverage for the code